operations/README.md

```diff
@@ -78,12 +78,22 @@ Build and customize image recipes with the Image Management Service (IMS).
 - [Import External Image to IMS](image_management/Import_External_Image_to_IMS.md)
 - [Import NCN Image to IMS](image_management/Import_NCN_Image_to_IMS.md)
 - [Customize an Image Root Using IMS](image_management/Customize_an_Image_Root_Using_IMS.md)
+<<<<<<< HEAD
```

Comment:
The backport looks like it needs to be re-examined. Although I think you can do an auto-backport if you just remove the linting changes to the workflow files in the 1.6 PR.